src/test/java/CalculatorTest.java

```diff
@@ -9,36 +9,68 @@
 public class CalculatorTest {
 
     @Test
-    public void test1() {
-        Integer rez  = Calculator.count('+', 2, 3);
-        Assert.assertEquals(rez, (Integer) 5);
+    public void testSum() {
+        Double rez = Calculator.count('+', 2.0, 3.0);
+        Double actualResult = 5.0;
+        Assert.assertEquals(rez, actualResult);
     }
 
     @Test
-    public void test2() {
-        Integer rez  = Calculator.count('*', 2, 2);
-        Assert.assertEquals(rez, (Integer) 4);
+    public void testMultiplication() {
+        Double rez = Calculator.count('*', 2.0, 2.0);
+        Double actualResult = 4.0;
+        Assert.assertEquals(rez, actualResult);
     }
 
     @Test
-    public void test3() {
-        Integer rez  = Calculator.count('-', 2, 2);
-        Assert.assertEquals(rez, (Integer) 0);
+    public void testMinus() {
+        Double rez = Calculator.count('-', 2.0, 2.0);
+        Double actualResult = 0.0;
+        Assert.assertEquals(rez, actualResult);
     }
 
     @Test
-    public void test4() {
-        Integer rez  = Calculator.count('/', 8, 4);
-        Assert.assertEquals(rez, (Integer) 2);
+    public void testDivision() {
+        Double rez = Calculator.count('/', 8.0, 4.0);
+        Double actualResult = 2.0;
+        Assert.assertEquals(rez, actualResult);
     }
 
-    @Test(expected = MyException.class)
-    public void test5() {
-        Calculator.count('$', 2, 2);
+    @Test(expected = NoCorrectMathOperationException.class)
+    public void testNoCorrectMathOperation() {
+        Calculator.count('$', 2.0, 2.0);
     }
 
     @Test(expected = NullPointerException.class)
-    public void test6() {
-        Calculator.count('-', null, 2);
+    public void testActWithNull() {
+        Calculator.count('-', null, 2.0);
+    }
+
+    @Test()
```

Comment:
```suggestion
    @Test
```
повидаляй скрізь дужки круглі

Reply:
видалив